src/utils/db.jsx

```diff
@@ -2,36 +2,46 @@ import axios from "axios";
 
 const api = axios.create({ baseURL: "https://nc-api-project.onrender.com/api" });
 
-async function getArticles() {
-  const result = await api.get("/articles");
+export async function getArticles(topic) {
```

Comment:
Take a look at the axios docs for a nicer way of passing option queries through :)